doc/adr/0003-use-elasticsearch.md

```diff
@@ -0,0 +1,21 @@
+# 3. Use ElasticSearch for Data Storage and Search
+
+Date: 2018-03-20
+
+## Status
+
+Accepted
+
+## Context
+
+Elasticsearch is configured as a cluster for reliability and failover, and provides a single point for data updates.
+Elasticsearch is also useful for filtering, especially when we will have the same source of data that need bucketing
+
+## Decision
+
+sexual-health-service finder will consume data from Elasticsearch rather than other sources.
```

Comment:
Can this be changed to something along the lines of `The application will use Elasticsearch for service data retrieval purposes.`